Readability.js

```diff
@@ -119,6 +119,36 @@ Readability.prototype = {
   },
 
   /**
+   * Iterate over a NodeList, which doesn't natively fully implement the Array
+   * interface.
```

Comment:
I'm always prepending test suite run calls with the `time` exec, I didn't notice any performance impact. I think js engines are now optimizing this properly, and even if it doesn't perform as good I think more readable code is worth the price.